scripts/init-database.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import yaml
+import sqlalchemy
+
+values = list()
+
+
+def retreiveValues():
+    stream = open("../.config/bassa/config.yml", "r")
+    docs = yaml.load_all(stream)
+    global values
+    for doc in docs:
+        for k, v in doc.items():
+            for m, n in v.items():
+                values.append(n)
+
+
+def createDatabase():
+    param = values[5]+'://'+values[0]+':'+values[4]+'@'+values[2]
+    engine = sqlalchemy.create_engine(param)
+    engine.execute("CREATE DATABASE IF NOT EXISTS " + values[3])
+
+
+def importSQL():
+    param = values[5]+'://'+values[0]+':'+values[4]+'@'+values[2]+'/'+values[3]
+    engine = sqlalchemy.create_engine(param)
+    fd = open("../Bassa.sql", 'r')
```

Comment:
Same for this about getting the path. 

And the purpose of this script is to create a new database with the given db name if it does not exist, create a user with the with the given username and password if it does not exist and then grant the correct access right to the user to the Bassa database.

Finally, we can import the Bassa.sql.